src/Components/Navbar.js

```diff
@@ -1,25 +1,61 @@
 import './Navbar.css'
 
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
 import PropTypes from 'prop-types'
 
+import { Link } from 'react-router-dom'
+
 import { Menubar } from 'primereact/menubar'
 
 function Navbar({ items, start, end }) {
+  const [version, setVersion] = useState('')
+
+  useEffect(() => {
+    fetch('/app.json')
+      .then((response) => response.json())
+
+      .then((data) => setVersion(data.version))
+
+      .catch((error) => {
+        console.log('Navbar useEffect', error)
+
+        alert('An error occurred please try again later.')
+      })
+  }, [])
+
   if (!end) {
     end = (
-      <a href="https://github.com/EddieHubCommunity/LinkFree" aria-label="View source on GitHub">
-        <i className="pi pi-github" style={{ fontSize: '2em' }}></i>
-      </a>
+      <div className="p-d-flex p-jc-center p-ai-center p-pr-2">
+        <Link
+          to={{ pathname: 'https://github.com/EddieHubCommunity/LinkFree' }}
```

Comment:
I **think** this should be a **a** tag

Reply:
ok, I will change it 👍

Reply:
Link is the correct way of doing it in react.

Reply:
Technically a tags work as well they just cause a whole page rerender which in this case doesn't matter as we're being directed to another site but it's best to use link for all links in a react project.

Reply:
Why was this a `a` tag earlier though 😅

Reply:
@krishguptadev We don't always get it right first time which is why it's good to collaborate on these things and make the code better. Like I said, an a tag will work and because it redirects outside the app it doesn't really cause too much of an issue here but a Link tag is actually what should be used in react so it was well spotted by @FidalMathew

Reply:
I noticed that the Link tag was used in the footer component. I knew that Link was better while working with react, so I changed it.😊